sys/net/network_layer/ng_ipv6/netif/ng_ipv6_netif.c

```diff
@@ -421,6 +422,119 @@ ng_ipv6_addr_t *ng_ipv6_netif_find_best_src_addr(kernel_pid_t pid, const ng_ipv6
     return _match_prefix(pid, dest, true);
 }
 
+/* TODO: put this somewhere more central and L2 protocol dependent */
+#define IID_LEN (8)
+
+static bool _hwaddr_to_iid(uint8_t *iid, const uint8_t *hwaddr, size_t hwaddr_len)
```

Comment:
Regardless of this issue (which is pretty complex, because it also includes how to auto-initialize addresses in general) we should really talk about this option, especially with regard to [substitutable features in 6LoWPAN ND](https://tools.ietf.org/html/rfc6775#section-1.4) which I'm currently working on. Are you at the Institute or the Hack'n'ACK today?